### ISSUE TITLE
Can not bring up Jupyter Notebook

### ISSUE THREAD

**Issue by myqq0000** [kubeflow/kubeflow#672]
Hi there.
I meet a question as mentioned in this question #554.

I follow the latest [user_guide](https://github.com/kubeflow/kubeflow/blob/master/user_guide.md#bringing-up-a-jupyter-notebook) to set up kubeflow in my own local server (Not on cloud or VM).
I have tried it a half of month ago. Because of the PVC problem can't set up JupyterHub.

And a few days before I found the document is updated. So I tried it again.
But after I clicked the "spawn" button, it not redirected to the Jupyter Notebook interface. Just stay at the dialog screen...

I try to find some information in kubernetes and find it be killed as this issue...
So how to solve it? Or is there any way to find more information of it?
I'm new to Kuberentes and Docker. So I can't understand it very well.
Thank you for your patient answers.

**Comment by pdmack** [comment]
Could you **attach** relevant segments of your docker daemon log where the kill appears to be happening please?

**Comment by myqq0000** [comment]
@pdmack So sorry for the late reply.
I use `sudo journalctl -fu docker.service`  to show recent log of docker daemon.
And after I tried to spawn it, there have been some log as below:

> 4月 21 15:38:12 ultron-PCN dockerd[13113]: time="2018-04-21T15:38:12.266422512+08:00" level=info msg="No non-localhost DNS nameservers are left in resolv.conf. Using default external servers: [nameserver 8.8.8.8 nameserver 8.8.4.4]"
> 4月 21 15:38:12 ultron-PCN dockerd[13113]: time="2018-04-21T15:38:12.266466131+08:00" level=info msg="IPv6 enabled; Adding default IPv6 external servers: [nameserver 2001:4860:4860::8888 nameserver 2001:4860:4860::8844]"
> 4月 21 15:38:12 ultron-PCN dockerd[13113]: time="2018-04-21T15:38:12+08:00" level=info msg="shim docker-containerd-shim started" address="/containerd-shim/moby/0741b2cc9378f749cea1729cc86af0bc3f3045fd88dec67ff9e6631857a5639c/shim.sock" debug=false module="containerd/tasks" pid=5394
> 4月 21 15:38:13 ultron-PCN dockerd[13113]: time="2018-04-21T15:38:13+08:00" level=info msg="shim docker-containerd-shim started" address="/containerd-shim/moby/59c73c45b40b65e54dbd132727543ced0f0f3dcdd71f938e2c135bab9a75f3de/shim.sock" debug=false module="containerd/tasks" pid=5555
> 4月 21 15:38:47 ultron-PCN dockerd[13113]: time="2018-04-21T15:38:47+08:00" level=info msg="shim reaped" id=59c73c45b40b65e54dbd132727543ced0f0f3dcdd71f938e2c135bab9a75f3de module="containerd/tasks"
> 4月 21 15:38:47 ultron-PCN dockerd[13113]: time="2018-04-21T15:38:47.233739603+08:00" level=info msg="ignoring event" module=libcontainerd namespace=moby topic=/tasks/delete type="*events.TaskDelete"
> 4月 21 15:38:47 ultron-PCN dockerd[13113]: time="2018-04-21T15:38:47+08:00" level=info msg="shim reaped" id=0741b2cc9378f749cea1729cc86af0bc3f3045fd88dec67ff9e6631857a5639c module="containerd/tasks"
> 4月 21 15:38:47 ultron-PCN dockerd[13113]: time="2018-04-21T15:38:47.654608268+08:00" level=info msg="ignoring event" module=libcontainerd namespace=moby topic=/tasks/delete type="*events.TaskDelete"

I can't extract useful information from it. Do you have any ideas about it? 
Thank you a lot for your timely reply~ 😊

**Comment by jlewi** [comment]
@myqq0000 Any luck?

**Comment by myqq0000** [comment]
@jlewi  Thank you a lot for your attention to my issue.
I am sorry that I have some other work to do so I ignore this issue these days.

I found that the kubeflow has updated to v0.1.2 today. So I try to deploy it again.
But it's still the same error.

I have tried this command but it can't return anything else than 

> Error from server (NotFound): pods "jupyter-test" not found

But I can found my pod in the dashboard of kubernetes in about 30 seconds. (And then it's killed.)
These are the events I copy from the dashboard:
> Events
> Message
> Source
> Sub-object
> Count
> First seen
> Last seen
> Successfully assigned jupyter-test to ultron-pcn
> default-scheduler
> -
> 1
> 2018-04-25T06:48 UTC
> 2018-04-25T06:48 UTC
> MountVolume.SetUp succeeded for volume "no-api-access-please"
> kubelet ultron-pcn
> -
> 1
> 2018-04-25T06:48 UTC
> 2018-04-25T06:48 UTC
> Container image "gcr.io/kubeflow-images-staging/tensorflow-1.7.0-notebook-cpu:v20180403-1f854c44" already present on machine
> kubelet ultron-pcn
> spec.containers{notebook}
> 1
> 2018-04-25T06:48 UTC
> 2018-04-25T06:48 UTC
> Created container
> kubelet ultron-pcn
> spec.containers{notebook}
> 1
> 2018-04-25T06:48 UTC
> 2018-04-25T06:48 UTC
> Started container
> kubelet ultron-pcn
> spec.containers{notebook}
> 1
> 2018-04-25T06:48 UTC
> 2018-04-25T06:48 UTC
> Killing container with id docker://notebook:Need to kill Pod
> kubelet ultron-pcn
> spec.containers{notebook}
> 1
> 2018-04-25T06:48 UTC
> 2018-04-25T06:48 UTC

Thank you for your attention again. If you have any advice please tell me and I will try it as soon as possible. :smiley:

**Comment by myqq0000** [comment]
@jlewi Oh I forget to say how am I running kubernetes.

I used kubeadm to run my kubernetes.
The versions of kubeadm, kubelet and kubectl are all v1.9.0.
And my kubernetes version is v1.9.6.

And I have set up the nvidia-docker and other plugins. (I want to use the GPU version of tensorflow at first.)
And I can see 2 GPU with the command `kubectl get nodes "-o=custom-columns=NAME:.metadata.name,GPU:.status.allocatable.nvidia\.com/gpu"`

Maybe it will have some affect? But I think it can't effect the CPU version...

Thanks for your reply~ 😃
